my-app/src/components/registration/RegistrationForm.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import "../registration/registrate.css";
+import { RegistrationMessage } from "./RegistrationMessage";
+
+export function RegistrationForm() {
+  const [form, setRegistrateForm] = useState({
+    password: "",
+    email: "",
+    name: "",
+  });
+  const [isEnter, setState] = useState(false);
```

Comment:
If you declare state variable with name `isEnter` then you have to name function `setIsEnter` according naming convention of React hooks

Reply:
Fixed